.gitignore

```diff
@@ -22,4 +22,21 @@ specification_compliance_report.html
 */**/.idea/
 /compliance
 /.smithy.lsp.log
-*/**/Folder.DotSettings.user
\ No newline at end of file
+*/**/Folder.DotSettings.user
+
+# These files are not tracked.
+# The release will force-commit these files, then the post-release commit will force-remove them.
+# Scripts at /scripts/release will run on all files between START_RELEASE_IGNORED_FILES and END_RELEASE_IGNORED_FILES.
+# START_RELEASE_IGNORED_FILES
+**/*py.dtr
+# Ignore everything in ImplementationFromDafny-go...
+**/ImplementationFromDafny-go/**
```

Comment:
You might also want to ignore `TestsFromDafny-go` ?

Reply:
Got it. Should this always be ignored? Or should we also commit this folder on releases?